src/main/webapp/app/components/panels/CompanionDiagnosticDevicePanel.tsx

```diff
@@ -42,25 +44,36 @@ const CompanionDiagnosticDevicePanel: React.FunctionComponent<StoreProps> = prop
   const createBiomarkerAssociation = (e: any) => {
     e.preventDefault();
     const association: Association = {
-      fdaSubmissions: fdaSubmissionValue.map(fdaSubmission => {
+      fdaSubmissions: fdaSubmissionValue?.map((fdaSubmission): FdaSubmission => {
         return {
-          id: fdaSubmission.value,
+          // TYPE-ISSUE: value is not on FdaSubmission
+          id: (fdaSubmission as any).value,
```

Comment:
I think in `FdaSubmissionSelect` component, we need a type for `FdaSubmissionSelectOptions` with `value` and `label`.

The type currently is set to `IFdaSubmission`, which is not the correct type